Misc/NEWS.d/next/Library/2020-05-19-01-40-51.bpo-40676.yJfq1J.rst

```diff
@@ -0,0 +1 @@
+Convert :mod:`csv` to use Argument Clinic for :func:`csv.field_size_limit`, :func:`csv.get_dialect`, :func:`csv.unregister_dialect` and :func:`csv.list_dialects`.
```

Comment:
```suggestion
Convert :mod:`csv` to use Argument Clinic for :func:`csv.field_size_limit`,
:func:`csv.get_dialect`, :func:`csv.unregister_dialect` and :func:`csv.list_dialects`.
```
We gained a new docs check that enforces a newline at the end of the NEWS fragment. Let's see if I can get GH suggestions to fix that.